### ISSUE TITLE
Weather Forecast

### ISSUE THREAD

**Issue by Hillehinsberg** [infokujur/Estonian-Open-Data-Issue-Tracker#5]
5 day-forecasts of temperature, precipitation and wind. Forecasts have to be provided for several regions in the country.

Temperature extremes
Temperature average
Wind speed
Wind direction
Precipitation Amount
Precipitation Probability
Forecast for current day and four following days

http://global.survey.okfn.org/dataset/weather

**Comment by tammelv** [comment]
Kas see siin ei ole juba? http://www.ilmateenistus.ee/teenused/ilmainfo/eesti-prognoos-xml/

**Comment by McSim** [comment]
No, it must be something like this - http://meteo.physic.ut.ee/et/archive.php
But not only for Tartu

**Comment by fassko** [comment]
I have made a small library which scrapes data from http://meteo.physic.ut.ee for Swift https://github.com/fassko/TartuWeatherProvider

And I created app for iOS https://itunes.apple.com/ai/app/meteo-tartu/id1082204986?mt=8 and also macOS https://itunes.apple.com/ee/app/meteo-tartu/id1082225005?mt=12 operating systems.

Currently I'm also making Apple Watch version of it.

Cheers!

**Comment by infokujur** [comment]
Dropping this link here: https://www.kaggle.com/berkeleyearth/climate-change-earth-surface-temperature-data

I hope its useful.

**Comment by infokujur** [comment]
http://www.ilmateenistus.ee/teenused/ilmainfo/eesti-prognoos-xml/
@McSim Whats missing here? Forecast xml is there.

**Comment by infokujur** [comment]
Riigi Ilmateenistuse andmeid ei ole https://opendata.riik.ee/teabevaldajad/ portaalis. Palume viidata andmed portaali.
